### ISSUE TITLE
Unpublished caches are seen at geopath page

### ISSUE THREAD

**Issue by kojoty** [opencaching/opencaching-pl#550]
z forum: 

> autor: areckis » środa, 20 kwietnia 2016, 08:50
> 
> Przy podglądzie GŚ widać na liście skrzynki nieopublikowane, ale co ważniejsze widać je również na mapce. Po powiększeniu mapy już ich nie ma, GPX ich również nie pobiera i do podglądu się nie można dostać. Ale chyba na małej mapce też nie powinny się pojawiać?
> 
> /Edit:
> Tutaj przykład: [http://opencaching.pl/powerTrail.php?ptAction=showSerie&ptrail=434]

**Comment by kojoty** [comment]
Łza. przypisuję do Ciebie bo ty tu expert - to jest zamierzone z jakiegoś powodu?

pozdrowienia.

**Comment by wloczynutka** [comment]
bug znany od jakiegoś czasu, jest już issue na ten temat.

**Comment by deg-pl** [comment]
Moja wypowiedź na forum:
Trzeba się tylko zastanowić, czy tak właśnie nie ma to funkcjonować.
Bo jadąc na jakąś ścieżkę chciałbym wiedzieć, czy za chwilę nie pojawi się jakiś nowy kesz na niej. Już kiedyś tak miałem, że zrobiłem sporo kilometrów by zdobyć ścieżkę, a gdy wróciłem domu okazało się, że właśnie został opublikowany nowy kesz.
Z drugiej strony - to, że nie ma podglądu i GPX nie pobiera, to chyba dobrze ;)

PS: Ścieżka podana jako przykład generalnie nie powinna być opublikowana. Bo w momencie jej publikacji jest niezdobywalna. I nad tym trzebaby się było zastanowić.

**Comment by wloczynutka** [comment]
Tak, Tak, myślę że tego policjanta który generował listę niezdobywalnych skrzynek trzeba przerobić tak żeby walił je pałką zamiast generować tylko listę. No może jakoś kulturalnie walił, żeby nie było że totalitaryzm wprowadzamy.

**Comment by kojoty** [comment]
...aaaa bo ta ścieżka jest opublikowana! Nie ogarnąłem tego :)

Nie no to jak ktoś tak robi, to zgadzam się, jest sam sobie winien i nie ma buga...

BTW. @wloczynutka gdzie jest to drugie issue na to?

**Comment by wloczynutka** [comment]
Jak będę miec chwilę to wykopię. Ale chyba teraz rozumiem, w tamtym issue chodziło o podobny problem, ale chyba jednak inny - gooŚcieżki umożliwiają dodanie skrzynek niezakceptowanych jeszcze przez COG.

natomiast to, że da się opublikowac niezdobywalną geoŚcieżkę, to też bug.

**Comment by deg-pl** [comment]
Pomysły z forum:
- Ranwers:
> może wyświetlać lokalizację kesza nieopublikowanego z błędem? Przesunięciem rzędu 200-500m. Wtedy wiadomo, że coś się tam czai, ale bez dokładnych współrzędnych. Odnośnie niewidoczności, to skoro kesz trafił na ścieżkę, to i tak informacja o nim już "poszła w świat"

- Verdant90:

> 
> Nie wiem jak Wam, ale najlogiczniejszym rozwiązaniem wydaje mi się takie:
> 1. Jeśli skrzynka jest "Jeszcze niedostępna", nikt poza Autorem kesza (no i oczywiście sferą COGową) go nie widzi, gdziekolwiek.
> 2. Jeśli Autor skrzynki umieszcza "jeszcze niedostępną" skrzynkę na geościeżce, to jest ona widoczna tylko dla Autora geościeżki i Autora skrzynki (które to osoby jak wcześniej pisałem mogą się sobie równać :) ) 

- deg:
> A może prościej- skrzynki (nieopublikowane) pokazują się na liście,ale nie są widoczne na mapie? To chyba rozwiązuje wszystkie powyższe problemy i jest łatwe w implementacji.

- Brasia:
> Podobnie z mapy GS powinny zniknąć wszystkie tymczasowo niedostępne i zarchiwizowane.

Będę stronniczy i napiszę, że dla mnie dwa ostatnie pomysły mają największy sens. Nie ma to jak pochwalić swój własny pomysł :D
